embuilder.py

```diff
@@ -33,7 +33,6 @@
         libcxx_noexcept
         libcxxabi
         gl
-        struct_info
```

Comment:
correct, see https://github.com/kripken/emscripten/commit/0822b148ecda0ce603f47d4e266395e951f06454 basically, bootstrapping is hard with a new backend like wasm. there is really no strong reason to bootstrap it on each dev's machine, since the output is portable anyhow, and can just be placed in the repo.